src/features/hotspots/setup/HotspotTxnsProgressScreen.tsx

```diff
@@ -77,15 +79,20 @@ const HotspotTxnsProgressScreen = () => {
 
     // check if add gateway needed
     const isOnChain = await hotspotOnChain(hotspotAddress)
-    const onboardingRecord = await getOnboardingRecord(hotspotAddress)
-    if (!onboardingRecord) return
+
+    let onboardingRecord
+    try {
+      onboardingRecord = await getOnboardingRecord(hotspotAddress)
+    } catch (error) {}
```

Comment:
Why are we swallowing the error here? We don't want to notify sentry or take any other action?

Reply:
Gotcha, please add comments explaining this.